mlflow/protos/service.proto

```diff
@@ -982,6 +982,25 @@ service MlflowService {
     };
   }
 
+  // Query aggregated metrics for traces, spans, or assessments.
+  // Supports grouping by dimensions and time-based aggregation.
+  rpc queryTraceMetrics(QueryTraceMetrics) returns (QueryTraceMetrics.Response) {
+    option (rpc) = {
+      endpoints: [
+        {
+          method: "POST"
+          path: "/mlflow/traces/metrics"
+          since: {
+            major: 3
+            minor: 0
```

Comment:
Isn't this 8 since our next release is 3.8? But it seems minor is usually specified as 0

Reply:
~This should mean whether it's mlflow 3.x, and yeah seems we never specify the minor version~
Actually, our mlflow server doesn't have different routes on minor versions so all the endpoints are either 2.0 or 3.0

Reply:
I see, of course this is out of scope of this pr, but it is confusing to always have minor=0 in protos.

Reply:
I think endpoint path is not impacted by the minor version here since we hardcode `.0` in https://github.com/mlflow/mlflow/blob/50f64943482855a762aee2d85927ee6de9c7811e/mlflow/server/handlers.py#L3816-L3817. So minor version here should be just a history logging purpose, and I guess there's no harm to make this accurate (again, not a scope of this pr).

Reply:
Yeah make sense :)